api/models/user_program.py

```diff
@@ -7,10 +7,6 @@
 class UserProgramQuery(BaseQuery):
     def upsert_user_program(self, user_id, data):
         user_program_details = self.get_latest_active_user_program(user_id)
-
-        if not user_program_details:
-            user_program_details = self.get_latest_user_program(user_id)
-
```

Comment:
Why removed this?

Reply:
Now there is no use of this condition. This is the previous condition where we are getting latest user program but now we need to get the latest active {IN-PROGRESS} user program in user program detail variable.